app/controllers/katello/api/v2/alternate_content_sources_controller.rb

```diff
@@ -2,8 +2,10 @@ module Katello
   class Api::V2::AlternateContentSourcesController < Api::V2::ApiController
     include Katello::Concerns::FilteredAutoCompleteSearch
 
-    before_action :find_authorized_katello_resource, :only => [:show, :update, :destroy]
-    before_action :find_smart_proxies
+    acs_wrap_params = AlternateContentSource.attribute_names.concat([:smart_proxy_ids])
+    wrap_parameters :alternate_content_source, include: acs_wrap_params
```

Comment:
Why are we doing this wrap_parameters?

Reply:
I removed the before_action because only create used it.  The wrap_parameters were introduced because the smart_proxy_ids aren't actually an attribute of ACSs, so I was having trouble permitting them as parameters.

Reply:
before_action :find_smart_proxies, only => [:create]